### PR TITLE
28-add-logo-animation-detail-page

### LINKED ISSUE #28: Add a subtle logo animation, detail page, on hover
Given that I'm on a Detail page of a project
When I hover over the Modus logo on the top of the page
Then I want to see a subtle animation so I know it's actionable

### PR DESCRIPTION
**Pull request by alexserb13**
Closes #28

### CODE REVIEW

File: src/components/Logo/AnimatedLogo.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+export const AnimatedLogo = props => {
+  const { s, classnames } = props;
```
[the remark this answers is not in the document]
Comment (alexserb13) [reply src/components/Logo/AnimatedLogo.js]: done

File: src/components/Logo/AnimatedLogo.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import s from './styles.module.scss';
+
+export const AnimatedLogo = props => {
+  const { classnames } = props;
```
Comment (grgur) [inline src/components/Logo/AnimatedLogo.js]: you should import `classnames` library, not pass it through props

File: src/components/Logo/LogoBlack.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 import classnames from 'classnames';
-import CloudinaryImage from '../CloudinaryImage';
+import { AnimatedLogo } from './AnimatedLogo';
 
 import s from './styles.module.scss';
 
 const LogoBlack = ({ className }) => (
   <div className={classnames(s.logoContainerBlack, className)}>
-    <CloudinaryImage filename="logo-black.svg" className={s.logoBlack} alt="Modus Labs" />
+    <AnimatedLogo s={s} classnames={classnames} />
```
Comment (grgur) [inline src/components/Logo/LogoBlack.js]: it looks like passing `s` property is useless here. I suggest leaving it out together with `classnames`

File: src/components/Logo/AnimatedLogo.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      <svg
+        width="68"
+        height="50"
+        xmlns="http://www.w3.org/2000/svg"
+        className="svg-logo"
+        viewBox="-105 -10 420 140"
+      >
+        <g transform="translate(-1 -1)" fill="none" fillRule="evenodd">
+          <path
+            className={classnames(s.rightfill)}
```
Comment (grgur) [inline src/components/Logo/AnimatedLogo.js]: Using `classnames` only makes sense if you programmatically construct CSS classes. This files doesn't do that. You should be able to use static class names and work with CSS.

File: src/components/Logo/AnimatedLogo.js (hunk 2 above)
Comment (alexserb13) [reply src/components/Logo/AnimatedLogo.js]: Will do

File: src/components/Logo/LogoBlack.js (hunk 3 above)
Comment (alexserb13) [reply src/components/Logo/LogoBlack.js]: On it

File: src/components/Logo/AnimatedLogo.js (hunk 4 above)
Comment (alexserb13) [reply src/components/Logo/AnimatedLogo.js]: Should i use template literals for all the variables ?

Comment (grgur) [reply src/components/Logo/AnimatedLogo.js]: No sir, CSS should be enough 😄 Even SCSS is ok 😄 

```jsx
<path className={s.myPath} />
```

```scss
.allPaths {
 fill: red;
}

.myPath {
  @extend .allPaths;
  stroke: blue;
}
```

Comment (alexserb13) [reply src/components/Logo/AnimatedLogo.js]: So using `classnames` to concatenate two classes is ok ?

**Comment by alexserb13** [comment]
What do you think? @grgur

**Pull request merged**